pkg/pipeline/transform/kubernetes/kubernetes.go

```diff
@@ -102,6 +103,7 @@ func (k *KubeData) GetInfo(ip string) (*Info, error) {
 			}
 
 			info.Owner = k.getOwner(info)
+			info.HostName = k.getHostName(info)
```

Comment:
You can move this under the `typePod` case because you output only if `HostIP` is not empty --- here: https://github.com/netobserv/flowlogs-pipeline/pull/196/files#diff-77b720060ce5b2e61d001e832eac1bf4d94ffdeb9734bbd4ee678d9398ad3892R155 

And this is set only for pods

Reply:
Yes I was wondering if we could do the same for `services` but it can run on multiple pods / hosts and we'll need to manage arrays.

Let's keep it simple for now, thanks for the feedback :+1: